docs/guides/upgrade/01_sdk-v1.mdx

```diff
@@ -0,0 +1,475 @@
+---
+id: sdk-v1
+title: Upgrade your Ory SDKs to v1
+sidebar_label: SDK V1 Upgrade Guide
+---
+
+# SDK V1 Upgrade Guide
+
+```mdx-code-block
+import CodeBlock from '@theme/CodeBlock'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+We are excited to announce the release of version 1.0 of Ory's Software Development Kits (SDKs) for all major languages. As part
+of releasing version 1.0, we have streamlined naming conventions and improved the API documentation.
+
+## History
+
+Ory SDKs have been around for a while and have gone through several transformations. We always relied on
+[openapi-generator](https://github.com/OpenAPITools/openapi-generator) to auto-generate SDKs from our OpenAPI specifications.
+
+## Backwards Compatibility
+
+Versions prior to 1.0 will remain functional and compatible with APIs offered on the Ory Network. However, we will no longer be
+maintaining these versions. We recommend that you upgrade to the latest version of the SDKs within the next 3 months.
+
+## Upgrade Guide
+
+Let's take a look at the changes we made to the SDKs and some examples of how to upgrade your code. We will use Golang for the
+examples, but the changes are similar for all languages.
+
+In essence, what is changing are naming conventions, not payloads or functionality. Let's create a new SDK client for the examples
+below:
+
+````mdx-code-block
+<Tabs>
+  <TabItem value="go" label="Golang" default>
+
+```shell
+go get github.com/ory/client-go@1.0.0
+```
+
+```go
+import ory "github.com/ory/client-go"
+
+var client = NewSDK()
+
+// Use this context to access Ory APIs which require an Ory API Key.
+var oryAuthedContext = context.WithValue(context.Background(), ory.ContextAccessToken, os.Getenv("ORY_API_KEY"))
+
+func NewSDK() *ory.APIClient {
+  conf := ory.NewConfiguration()
+  conf.Servers = ory.ServerConfigurations{{
+    URL: "https://{your-project}.projects.oryapis.com/",
+  }}
+
+  // If you want the SDK to use a custom HTTP Client, set:
+  //  conf.HTTPClient = &http.Client{ /* ... */ }
+
+  return ory.NewAPIClient(conf)
+}
+```
+
+  </TabItem>
+  <TabItem value="js" label="JavaScript">
+
+```shell
+npm i @ory/client@~1.0.0
+```
+
+```diff
+import {
+  Configuration,
+- V0alpha2Api
++ OAuth2Api,
++ IdentityApi,
++ FrontendApi,
+} from "@ory/client"
+
+const config = new Configuration({
+  basePath: "https://{your-project}.projects.oryapis.com/",
+  accessToken: process.env.ORY_API_KEY,
+  baseOptions: {
+    withCredentials: true, // Important for CORS
+    timeout: 30000, // 30 seconds
+  },
+})
+
+const ory = {
+- v0alpha2: V0alpha2Api
++ identity: new IdentityApi(config),
++ frontend: new FrontendApi(config),
++ oauth2: new OAuth2Api(config),
+}
```

Comment:
Should we add this code as manually maintained part to the Ory client? Looks like it belongs there...
We could just append that to the `index.ts`, it's content is pretty static so that should work easily https://github.com/OpenAPITools/openapi-generator/blob/510f79989a003d2c81e6960ef09e8ba3ee43843d/modules/openapi-generator/src/main/resources/typescript-axios/index.mustache

Reply:
Would be epic if that would be possible. Also creating the `withCredentails` thing and setting the SDK url. If this is easily possible it would be awesome @zepatrik

Reply:
https://github.com/ory-corp/cloud/issues/3401